quiz/web/vite.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+
+export default defineConfig({
+  server: {
+    port: 3000,
```

Comment:
port powinien zostać pobrany z env - https://nodejs.dev/en/learn/how-to-read-environment-variables-from-nodejs/